templates/template.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
I think we can do without this one, although I do like a consistent template for everything (including IDP etc. at a later stage). Shall we name it `main.html` or so?

Reply:
Main reason for it is to have the title and meta tags (like viewport). Could also be used in the future for linking a css file.

Reply:
Yeah I do like that bit!

Reply:
Yeah that's great indeed; we should strive to use it as a base for all later on.